src/api/AdjudicationAPI.js

```diff
@@ -0,0 +1,6 @@
+import { get } from './ApiUtils';
+import { getAdjudicationsURL } from './PathConstants';
+
+export const getAdjudications = (id) => {
+    return get(getAdjudicationsURL(id));
```

Comment:
let's replace `getAdjudicationsURL` with the actual string and remove it from `PathConstants.js`